osu.Game/OsuGame.cs

```diff
@@ -1023,9 +1010,13 @@ protected override void UpdateAfterChildren()
         {
             base.UpdateAfterChildren();
 
-            screenOffsetContainer.Padding = new MarginPadding { Top = ToolbarOffset };
+            ScreenOffsetContainer.Padding = new MarginPadding { Top = ToolbarOffset };
             overlayContent.Padding = new MarginPadding { Top = ToolbarOffset };
 
+            var settingsOffset = Settings.HorizontalScreenOffset * SCREEN_OFFSET_RATIO;
+            var notificationsOffset = Notifications.HorizontalScreenOffset * SCREEN_OFFSET_RATIO;
```

Comment:
This whole song and dance with adding `HorizontalScreenOffset` and making everything declare it and move the contents around is weird to me. Wouldn't something like using `ScreenSpaceDrawQuad` or similar replace the need for doing any of this entirely?

Reply:
Hmm, will have to fix `SettingsOverlay` not autosizing to its content properly first (if that's possible), then I could consider that

Reply:
Actually `SettingsOverlay` has always been autosizing correctly, not sure what I was seeing at first.

Reply:
While it does shorten things significantlly, the way `SettingsOverlay` is currently structured makes it impossible to match exact behaviour:

https://user-images.githubusercontent.com/22781491/128606435-4edeebf0-4854-4bfc-b8ca-67d322e8dfa5.mp4

Although I'm *very slightly* tempted to leave that as a cool effect :p (would definitely need more opinions)

https://user-images.githubusercontent.com/22781491/128606563-785b0116-760b-4b14-999d-fa676697f627.mp4

Reply:
I think it looks okay.